docs/tutorial-dev-kinto-admin.rst

```diff
@@ -0,0 +1,156 @@
+
+Prerequisites
+-------------
+
+This guide assumes you have already installed and set up the following:
+
+- :ref:`tutorial-local-server` with multi-signoff
+
+Kinto Admin
+-----------
+
+This part is very classic. We recommend `using NVM <https://github.com/nvm-sh/nvm>`_ in order to have recent versions of Node and NPM.
+
+.. code-block::
+
+    git clone git@github.com:Kinto/kinto-admin.git
+
+    cd kinto-admin/
+
+    npm install
+    npm start
+
+The UI should be accessible at http://0.0.0.0:3000
+
+Initialization script
+---------------------
+
+Since the container is not configured with a real database by default, the content is flushed on each restart.
+
+We'll create a small bash script ``init.sh`` with the following commands:
+
+.. code-block:: bash
+
+    set -e
+    set -o pipefail
+
+    SERVER=http://localhost:8888/v1
+
+In the :ref:`prerequisiste tutorial <tutorial-local-server>`, an ``admin`` user was created, as well as the basic buckets. Let's add that to our script:
+
+.. code-block:: bash
+
+    curl -X PUT --fail --verbose ${SERVER}/accounts/admin \
+         -d '{"data": {"password": "s3cr3t"}}' \
+         -H 'Content-Type:application/json'
+
+    BASIC_AUTH=admin:s3cr3t
+
+    curl -X PUT --fail --verbose ${SERVER}/buckets/main-workspace \
+         -d '{"permissions": {"collection:create": ["system.Authenticated"], "group:create": ["system.Authenticated"]}}' \
+         -H 'Content-Type:application/json' \
+         -u $BASIC_AUTH
+
+    curl -X PUT --fail --verbose ${SERVER}/buckets/main-preview \
+         -d '{"permissions": {"read": ["system.Everyone"]}}' \
+         -H 'Content-Type:application/json' \
+         -u $BASIC_AUTH
+
+    curl -X PUT --fail --verbose ${SERVER}/buckets/main \
+         -d '{"permissions": {"read": ["system.Everyone"]}}' \
+         -H 'Content-Type:application/json' \
+         -u $BASIC_AUTH
+
+In order to play with multi-signoff, we'll create an ``editor`` and a ``reviewer`` accounts, put these lines in the ``init.sh`` script.
+
+.. code-block:: bash
+
+    curl -X PUT --fail --verbose ${SERVER}/accounts/editor \
+         -d '{"data": {"password": "3d1t0r"}}' \
+         -H 'Content-Type:application/json'
+
+    curl -X PUT --fail --verbose ${SERVER}/accounts/reviewer \
+         -d '{"data": {"password": "r3v13w3r"}}' \
+         -H 'Content-Type:application/json'
+
+
+Now create a collection, with a dedicated reviewer group:
+
+.. code-block:: bash
+
+    curl -X PUT --fail --verbose ${SERVER}/buckets/main-workspace/collections/password-recipes \
+         -H 'Content-Type:application/json' \
+         -u editor:3d1t0r
+
+
+    curl -X PATCH --fail --verbose $SERVER/buckets/main-workspace/groups/password-recipes-reviewers \
+         -H 'Content-Type:application/json-patch+json' \
+         -d '[{ "op": "add", "path": "/data/members/0", "value": "account:reviewer" }]' \
+         -u $BASIC_AUTH
+
+And at last, create some records, request review and approve changes.
```

Comment:
This almost looks like literate code. Maybe we should just put all the code in a script and ship the script? Of course, then I would want to write the script in Python.

Reply:
The whole repo relies on `curl`.

And I thought it would make more sense to explain what we were doing here, instead of just `wget http://.../init.sh | bash`